src/site/content/en/blog/samesite-cookies-explained/index.md

```diff
@@ -268,68 +268,77 @@ on your site, you should also check with the provider that they are updating
 their services. You may need to update your dependencies or snippets to ensure
 that your site picks up the new behavior. {% endAside %}
 
-These changes are all backwards-compatible with other browsers. By making these
-changes to your cookies, you are making their intended use explicit rather than
-relying on the default behavior of the browser. Likewise, any clients that do
-not recognize `SameSite=None` as of yet should ignore it and carry on as if the
-attribute was not set.
+These changes are backwards-compatible with browsers that have correctly
```

Comment:
Language a little funky... what about 'that have correctly implemented earlier versions of the `SameSite` attribute or that do not'

Reply:
Hmm... looks like I lost part of the sentence somewhere! :O